Benachrichtigung/module.php

```diff
@@ -236,39 +244,39 @@ public function GetConfigurationForm()
                                 ]
                             ],
                             [
-                                'name'  => 'active',
+                                'name'    => 'active',
                                 'caption' => 'Active',
-                                'width' => '60px',
-                                'add'   => true,
-                                'edit'  => [
+                                'width'   => '60px',
+                                'add'     => true,
+                                'edit'    => [
                                     'type' => 'CheckBox'
                                 ]
                             ],
                             [
-                                'name'  => 'status',
+                                'name'    => 'status',
                                 'caption' => 'Status',
-                                'width' => '300px',
-                                'add'   => ''
+                                'width'   => '300px',
+                                'add'     => ''
                             ]
                         ],
                         'values' => $notificationValues
                     ],
                     [
-                        'type' => 'ExpansionPanel',
+                        'type'    => 'ExpansionPanel',
                         'caption' => 'Advanced Settings',
```

Comment:
Eigentlich nicht Teil deines Umbaus, magst du trotzdem die Advanced Settings mit in die locale.json aufnehmen?

Reply:
Wird im anderen Branch erledigt